spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb

```diff
@@ -4,10 +4,11 @@
 
 RSpec.describe Wakes::GoogleAnalyticsApiWrapper do
   describe '#get_page_of_pageviews' do
-    let(:rows) { [['/path', 151], ['/path/2', 500]] }
-    let(:results) { double('Results', :rows => rows, :items_per_page => 2) }
+    let(:row1) { double('Row', dimension_values: [double('DimensionValue', value: '/path')], metric_values: [double('MetricValue', value: 151)]) }
```

Comment:
@benhutton Can we use dummy paths like `/resources/message-1` and `/resources/article-1` as the dummy values here? That will help a reader get better oriented.

Reply:
done!